crypto/Cargo.toml

```diff
@@ -9,9 +9,12 @@ edition = "2021"
 openssl = ["dep:openssl", "dep:hkdf", "dep:sha2"]
 rustcrypto = ["dep:hkdf", "dep:hmac", "dep:p256", "dep:p384", "dep:rand", "dep:sha2", "dep:base64ct", "dep:ecdsa", "dep:sec1"]
 deterministic_rand = ["dep:rand"]
+no-cfi = []
 
 [dependencies]
 arrayvec = { version = "0.7.4", default-features = false, features = ["zeroize"] }
+caliptra-cfi-lib-git.workspace = true
+caliptra-cfi-derive-git.workspace = true
```

Comment:
The issue is that we would need to specify another feature to indicate that there is cfi and then add these dependencies only if that feature is enabled. But then it gets confusing since we have both a "cfi" feature and a "no-cfi" feature which might not be mutually exclusive. 

The other caliptra crates depending on cfi also do not make the cfi dependencies optional, so I think it's ok.